<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Log-in Biblioteca</string>
  </property>
  <property name="windowIcon">
   <iconset theme="system-search"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="NomeLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>36</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NomeInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius: 10px
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="SenhaLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>35</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Senha:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SenhaInput">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius: 10px}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                border-radius: 10px;
                padding: 10px;
                background-color: #7FF517;
                color: #ffffff;
            }

            QPushButton:hover {
                background-color: #71D716;
            }</string>
    </property>
    <property name="text">
     <string>Log-in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SenhaBtn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                border-radius: 10px;
                padding: 10px;
                background-color: #FF0000;
                color: #ffffff;
            }

            QPushButton:hover {
                background-color: #ED0000;
            }</string>
    </property>
    <property name="text">
     <string>Esqueci a senha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CriarContaBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
                border-radius: 10px;
                padding: 10px;
                background-color: #3498db;
                color: #ffffff;
            }

            QPushButton:hover {
                background-color: #2980b9;
            }</string>
    </property>
    <property name="text">
     <string>Criar Conta</string>
    </property>
   </widget>
   <widget class="QLabel" name="InputErro">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
